data/outport/outportBlock.proto

```diff
@@ -80,11 +80,39 @@ message ValidatorRatingInfo {
   float   Rating    = 2 [(gogoproto.jsontag) = "rating"];
 }
 
+message ValidatorsRating{
+  string   IndexID                                   = 1 [(gogoproto.jsontag) = "indexID"];
+  repeated ValidatorRatingInfo ValidatorsRatingInfo  = 2 [(gogoproto.jsontag) = "validatorsRatingInfo,omitempty"];
+}
+
 message RoundInfo {
-  uint64 Index                   = 1 [(gogoproto.jsontag) = "index"];
-  repeated uint64 SignersIndexes = 2 [(gogoproto.jsontag) = "signersIndexes"];
-  bool BlockWasProposed          = 3 [(gogoproto.jsontag) = "blockWasProposed"];
-  uint32 ShardId                 = 4 [(gogoproto.jsontag) = "shardId"];
-  uint32 Epoch                   = 5 [(gogoproto.jsontag) = "epoch"];
-  uint64 Timestamp               = 6 [(gogoproto.jsontag) = "timestamp"];
+  uint64          Index            = 1 [(gogoproto.jsontag) = "index"];
```

Comment:
this should be renamed in `Round`

Reply:
I've copied the fields as they were. It was `Index`: https://github.com/multiversx/mx-chain-core-go/blob/9b2d95bc7a05933c8be230ad6730faff8b4a0b2f/data/outport/dtos.go#L94

Shall we rename it?

Reply:
yes. it's ok to rename it because the name was not correct.